assignment/models/homework.py

```diff
@@ -22,11 +22,11 @@ class SchoolTeacherAssignment(models.Model):
     cmp_id = fields.Many2one('res.company', 'Company Name',
                              related='school_id.company_id')
 
-    @api.model
-    def default_get(self, fields_list):
-        res = super(SchoolTeacherAssignment, self).default_get(fields_list)
-        res.update({'state': 'draft'})
-        return res
+#    @api.model
+#    def default_get(self, fields_list):
+#        res = super(SchoolTeacherAssignment, self).default_get(fields_list)
+#        res.update({'state': 'draft'})
+#        return res
 
```

Comment:
Do we really need this method? We have set default=draft in field definition.
You must remove it.